releasenotes-builder/src/main/java/com/github/checkstyle/MainProcess.java

```diff
@@ -237,12 +240,14 @@ private static List<String> runPostPublication(CliOptions cliOptions) {
      * @param errors list of publication errors.
      */
     private static void runTwitterPublication(CliOptions cliOptions, List<String> errors) {
-        final TwitterPublisher twitterPublisher = new TwitterPublisher(
-            cliOptions.getOutputLocation() + TWITTER_FILENAME,
-            cliOptions.getTwitterConsumerKey(), cliOptions.getTwitterConsumerSecret(),
-            cliOptions.getTwitterAccessToken(), cliOptions.getTwitterAccessTokenSecret());
         try {
-            twitterPublisher.publish();
+            final String post = Files.readString(
+                    Paths.get(cliOptions.getOutputLocation() + TWITTER_FILENAME),
+                    StandardCharsets.UTF_8);
+
+            TwitterPublisher.publish(cliOptions.getTwitterConsumerKey(),
```

Comment:
Another example of the split. We separated out the reading of the post so we can eventually verify what is exactly being posted.